File: project.py
from project_module import project_object, image_object, link_object, challenge_object

p = project_object('image_fader', 'Image fader')
p.domain = 'http://www.aidansean.com/'
p.path = 'image_fader'
p.preview_image    = image_object('%s/images/project.jpg'   %p.path, 150, 250)
p.preview_image_bw = image_object('%s/images/project_bw.jpg'%p.path, 150, 250)
p.folder_name = 'aidansean'
p.github_repo_name = 'image_fader'
p.mathjax = True
p.tags = 'Images,Tools,Web design'
p.technologies = 'canvas,HTML,JavaScript'
p.links.append(link_object(p.domain, 'image_fader/', 'Live page'))
p.introduction = 'This is a simple script that fades an image to a given color and transparency.  It\'s used to make images for use in web design (for example, a background image that needs to fade to match the background color.)'
p.overview = '''This was made to edit the background images on my website.'''

p.challenges.append(challenge_object('Due to lack of time it only fades from the top at the moment.', 'The other three directions will be added as needed!', 'To be revisited.'))

File: doc/project.php
<?php
include_once($_SERVER['FILE_PREFIX']."/project_list/project_object.php") ;
$github_uri   = "https://github.com/aidansean/image_fader" ;
$blogpost_uri = "http://aidansean.com/projects/?tag=image_fader" ;
$project = new project_object("image_fader", "Image fader", "https://github.com/aidansean/image_fader", "http://aidansean.com/projects/?tag=image_fader", "image_fader/images/project.jpg", "image_fader/images/project_bw.jpg", "This is a simple script that fades an image to a given color and transparency.  It's used to make images for use in web design (for example, a background image that needs to fade to match the background color.)", "Images,Tools,Web design", "canvas,HTML,JavaScript") ;
?>